packages/jest-util/src/fake_timers.js

```diff
@@ -7,11 +7,11 @@
  * @flow
  */
 
-import type {ProjectConfig} from 'types/Config';
-import type {Global} from 'types/Global';
-import type {ModuleMocker} from 'jest-mock';
+import type {ProjectConfig } from 'types/Config';
```

Comment:
you should run `yarn lint --fix` to remove all of these whitespace changes

Reply:
I also doubt you want the `yarn.lock` changes

Reply:
hi, @SimenB , i get a lot of `Unable to resolve path to module` issues when `yarn lint --fix` ...and i also had to add `"eslint-config-fbjs": "^2.0.0"` to make `yarn lint` work. do i do something wrong here?

Reply:
remove your changes to yarn.lock, and run `yarn clean-all && yarn && yarn lint --fix`. Make sure you are on yarn@1.1.0 as well.

That should fix it 🙂

Reply:
hehe, i'll try, thanks! just switched from atom to vs code (there was some unexpected indent behaviour which i resolved now)

Reply:
```yarn lint v0.24.6
$ eslint . --cache --ext js,md
Cannot read config file: /Users/timoobereder/Documents/Projects/react/jest/packages/eslint-config-fb-strict/index.js
Error: Cannot find module 'eslint-config-fbjs'
Referenced from: /Users/timoobereder/Documents/Projects/react/jest/.eslintrc.js
Error: Cannot read config file: /Users/timoobereder/Documents/Projects/react/jest/packages/eslint-config-fb-strict/index.js
Error: Cannot find module 'eslint-config-fbjs'
Referenced from: /Users/timoobereder/Documents/Projects/react/jest/.eslintrc.js
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/timoobereder/Documents/Projects/react/jest/packages/eslint-config-fb-strict/index.js:10:20)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
/Users/timoobereder/.yarn/bin/yarn: line 31: 92774 Segmentation fault: 11  node "$basedir/yarn.js" "$@"```

Reply:
Segfault, interesting...

Reply:
i'm sure i have the latest yarn version through homebrew

Reply:
@DeFuex I pushed to your branch. Still a couple of lint errors, but those are actual issues, not style related.

So just pull, and do `yarn` and you should be good